### DISCUSSION TITLE
Renovate can't create onboarding PR due to GitHub IP allowlist

### DISCUSSION THREAD

**Discussion by andersfischernielsen** [renovatebot/renovate#13215] [Request Help]
I'm testing Renovate out for use in [Pleo](https://github.com/pleo-io), but I'm having issues even getting the app to run on a single repository. Renovate would simplify our current dependency management flow, so I'm keen to set it up 🚀 

After adding the Renovate GH app to our organisation and a single simple repository nothing happens, though.
We have a GitHub IP allowlist set up for our organisation for hardening and I have a suspicion that Renovate can't see and create PRs for the repositories we enable it on. 

So my question is then: 
Is there a range of IPs we can add to our IP allowlist to allow the GH App to get through to our organisation? 

I couldn't find any existing issues or discussion relating to the IP allowlist, hence this discussion.

Thanks for any help in advance!

**Comment by rarkins** [comment]
No, the app IP addresses are not fixed. You'll need to self host from your own fixed IP

**Reply by andersfischernielsen** [reply]
Ah, damn. Sad to hear that - but thanks for the quick response! 🙏🏼
